crypto/rand/rand_lib.c

```diff
@@ -46,7 +46,7 @@ void rand_read_tsc(RAND_poll_fn cb, void *arg)
     unsigned char c;
     int i;
 
-    for (i = 0; i < 10; i++) {
+    for (i = 0; i < TSC_READ_COUNT; i++) {
         c = (unsigned char)(OPENSSL_rdtsc() & 0xFF);
```

Comment:
On related note. Currently used `size_t OPENSSL_ia32_rd{seed|rand}(void)` were criticized, which is why `size_t OPENSSL_ia32_rd{seed|rand}_bytes(char *buf,size_t len)` were added.